WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -2338,6 +2338,7 @@ class MediaProgressCoordinator: NSObject {
         if let mediaUploadingProgress = self.mediaUploadingProgress, !isRunning {
             mediaUploadingProgress.removeObserver(self, forKeyPath: #keyPath(Progress.fractionCompleted))
             self.mediaUploadingProgress = nil
+            self.mediaUploading.removeAll()
```

Comment:
I added the same call on my legacy PR, but found out we cannot do this. 
If we remove all the previous upload you can have this scenario:

1. You upload some media that fails
2. After the first media fails, you upload another media that is successful
3. Now if you save the post it will not detect the first failed media of the post, because we remove all media on step 2.